### DISCUSSION TITLE
Ability to report and filter gross images

### DISCUSSION THREAD

**Discussion by Zero3K** [bluesky-social/atproto#1346] [Bluesky Lexicons]
It would be nice if Bluesky had the ability to report and filter gross images.

**Comment by Zero3K** [comment]
Any thoughts on this?

**Reply by bnewbold** [reply]
We have `disgusting` as a proposed label value in our proposed iteration on labeling:
https://github.com/bluesky-social/proposals/tree/main/0002-labeling-and-moderation-controls#unpleasant

Reporting has a smaller ontology of report types, but there is an option to include extra context in the report reason field. The "type" doesn't actually need to have any impact on the moderation review process, it is just a hint. That is, in the current reporting scheme reports bring moderator attention to content and the moderator compares the content against all policies, not just the policy indicated in the report type.

**Reply by Zero3K** [reply]
There's self-labeling but there's no option for the mentioned kind of images.

**Comment by Zero3K** [comment]
Any updates?
